community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/OuterHashJoin.scala

```diff
@@ -21,12 +21,17 @@ package org.neo4j.cypher.internal.v3_4.logical.plans
 
 import org.neo4j.cypher.internal.ir.v3_4.{CardinalityEstimation, IdName, PlannerQuery}
 
+/**
+  * Variant of NodeHashJoin. Also builds a hash table using 'left' and produces merged left and right rows using this
+  * table. In addition, also produces left and right rows with missing key values, and right rows that do not match
+  * in the hash table. In these additional rows, variables from the opposing stream are set to NO_VALUE.
+  */
```

Comment:
Maybe mention that this is a variation to what in relational algebra is known as an outer join?